aerospike-mixin/config.libsonnet

```diff
@@ -0,0 +1,24 @@
+{
+  _config+:: {
+    enableMultiCluster: true,
+    aerospikeSelector: if self.enableMultiCluster then 'job=~"$job", cluster=~"$cluster"' else 'job=~"$job"',
+    multiclusterSelector: 'job=~"$job"',
+
+    dashboardTags: ['aerospike-mixin'],
+    dashboardPeriod: 'now-1h',
```

Comment:
Any specific reason for 1h?

Reply:
I've always been told 1hr is standard.

Reply:
We normally use 30m. If you do not set this manually, the `util.prepare-dashboards` function set it to the default `now-30m`. This avoids uneeded long running queries.

Reply:
Good to know, I'll make sure to radiate this to the team.